src/tiled/tilesetdocument.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include "tilesetdocument.h"
 
+#include "fileformat.h"
```

Comment:
Already includes `tilesetformat.h`, so this is superfluous.

Reply:
Done